src/Persistence.php

```diff
@@ -40,7 +40,7 @@ public static function connect($dsn, $user = null, $password = null, $args = [])
         // Process DSN string
         $dsn = \atk4\dsql\Connection::normalizeDSN($dsn, $user, $password);
 
-        $driver = isset($args['driver']) ? strtolower($args['driver']) : $dsn['driver'];
+        $driver = strtolower($args['driver'] ?? $dsn['driver']);
```

Comment:
This is slight actual change, but I think that's ok.

Reply:
Step by step improvement ;-)